### PR TITLE
feat: apply google oauth

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [ ] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA

### CODE REVIEW

File: apps/web/src/services/auth/authenticator/external/GOOGLE_OAUTH2/authenticator/google-auth.ts [hunk 1]
```diff
@@ -46,23 +46,28 @@ class GoogleAuth extends Authenticator {
...
-                    await GoogleAuth.onSuccess(res.access_token);
-                    if (onSignInCallback) onSignInCallback();
+                    try {
+                        await GoogleAuth.onSuccess(res.access_token);
+                        if (onSignInCallback) onSignInCallback();
+                    } catch (e) {
+                        if (onErrorCallback) onErrorCallback(e, res.access_token);
+                    }
                 } else {
+                    console.log('callback error');
```
Comment (piggggggggy) [inline apps/web/src/services/auth/authenticator/external/GOOGLE_OAUTH2/authenticator/google-auth.ts:70]: Question: is this intended?

**Review by piggggggggy — APPROVED** [review]
LGTM

**Review by yuda110 — APPROVED** [review]
LGTM!!!

File: apps/web/src/services/auth/authenticator/external/GOOGLE_OAUTH2/authenticator/google-auth.ts (hunk 1 above)
Comment (skdud4659) [reply apps/web/src/services/auth/authenticator/external/GOOGLE_OAUTH2/authenticator/google-auth.ts:70]: removed it! :)

**Pull request merged**